CHANGELOG.md

```diff
@@ -52,6 +53,73 @@ RELEASE CHANGELOG TEMPLATE:
 ### Documentation
 -->
 
+<a name="v3.2.0"></a>
+# [3.2.0](https://github.com/ligato/vpp-agent/compare/v3.1.0...v3.2.0) (2020-10-XX)
+
+### COMPATIBILITY
+- VPP 20.09 (compatible)
+- VPP 20.05 (default)
+- VPP 20.01 (backwards compatible)
+- VPP 19.08 (backwards compatible)
+- ~~VPP 19.04~~ (no longer supported)
+
+### KNOWN ISSUES
+
+### BREAKING CHANGES
+
+### Bug Fixes
+- Fixes and improvements for agentctl and models (#1643)
```

Comment:
btw will it work without a link?
I see that previously it was like `[#1630](https://github.com/ligato/vpp-agent/pull/1630)`